options/posix/generic/sched-stubs.cpp

```diff
@@ -1,9 +1,10 @@
 
 #include <bits/ensure.h>
+#include <errno.h>
 #include <sched.h>
 
 #include <mlibc/debug.hpp>
-#include <mlibc/posix-sysdeps.hpp>
+#include <mlibc/all-sysdeps.hpp>
```

Comment:
Why this change? The posix option should only include the posix sysdeps, if posix needs a sysdep not in the header then it should be added.